server/webserver.go

```diff
@@ -750,6 +753,25 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 		w.Write([]byte(fmt.Sprintf("\n\nLatestPlaylist: %v", s.LatestPlaylist())))
 	})
 
+	mux.HandleFunc("/logLevel", func(w http.ResponseWriter, r *http.Request) {
+		if vFlag == nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		level := r.URL.Query().Get("level")
```

Comment:
Is it appropriate to use a query parameter here? We use form values elsewhere.

Reply:
Made it to be part of the path.
As said, it is meant to be used from `curl`, and post form values from `curl` is not very convenient.

Reply:
`curl -F loglevel=debug http://localhost:7935/setLogLevel` is pretty easy, no? I'd really rather we stay consistent with how we do things elsewhere in the CLI.

Reply:
> is pretty easy, no? 

No, it is required remembering of `curl` syntax for that case and parameter name, which I rather not to.

But OK, I've changed it to use form data.